api/net/Areas/Subscriber/Controllers/UserController.cs

```diff
@@ -4,16 +4,14 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Options;
 using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Subscriber.Models.User;
 using TNO.API.CSS;
 using TNO.API.Models;
-using TNO.DAL.Services;
-using TNO.API.Areas.Subscriber.Models;
-using TNO.Keycloak;
 using TNO.Core.Exceptions;
 using TNO.Core.Extensions;
+using TNO.DAL.Services;
 using TNO.Entities;
-using TNO.API.Areas.Admin.Models.User;
```

Comment:
@jdtoombs @fbarreta do not mix models.  Every controller has its own models so that we fully control what is being serialized.  Subscribers should never have access to the same data as administrators.  This is how private information gets leaked on apps that have private information.